src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -87,7 +87,7 @@ void setActivityStore(ActivityStore activityStore) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    List<Conversation> conversations = conversationStore.getAllConversations();
```

Comment:
`getAllConversations` is still there, I just added the new `getAllPublicConversations` function and used it

Reply:
So this just allows for public access?

Reply:
Ya, it only retrieves conversations that are not marked as private (so far being only chatbot conversations)